models/campaignModel.js

```diff
@@ -2,6 +2,7 @@ const db = require('../database/dbConfig');
 
 const CampUpdate = require('./updateModel.js');
 const CampComments = require('./commentsModel.js');
+const SkilledImpactRequests = require('./skilledImpactRequestsModel.js');
 
 function find() {
```

Comment:
This function is turning into callback hell. Can you please refactor with `async/await`?